docs/unified-test-documentation/dasharo-compatibility/31G-ec-and-superio.md

```diff
@@ -2158,6 +2158,78 @@ in the charger into the DUT when the battery level sits between 95% and 98%.
     not charging
     ```
 
+## ECR031.001 EC firmware sync in coreboot
+
+**Test description**
+
+This test verifies the automatic EC firmware update / sync feature in coreboot.
+
+**Test configuration data**
+
+1. `FIRMWARE` = Dasharo
+1. `OPERATING_SYSTEM` = Ubuntu 22.04
+
+**Test setup**
+
+1. Proceed with the
+   [Test cases common documentation](#test-cases-common-documentation) section.
+1. Downgrade to the previous firmware version (both EC and coreboot must be
+   downgraded).
+1. Keep AC adapter or USB-PD power supply connected for the duration of the
+   test.
+
+**Test steps**
+
+1. Power on the DUT.
+1. Proceed with the firmware update steps [as documented in the firmware update
+   guide](/guides/firmware-update).
+1. Reboot the DUT after updating firmware.
+1. Wait until the DUT updates the EC (indicated by the fans spinning at full
+   speed) and powers off.
```

Comment:
Personally I would be scared if my laptop spins the fans at full speed after a firmware update. DO we display something on the screen at least?

Reply:
We don't, because the update takes place before we can display anything. We could use the ADL SoL feature that was added specifically for this use case but that won't work for TGL models.

For now the idea was to show the information about EC update just before DTS reboots after an update.

Reply:
That is why I suggested to move the EC sync to EDK2. We will have to do so anyways if we want to support capsule updates

Reply:
Will be displayed on MTL using FSP-M SOL. Something similar will be enabled for ADL later.